Resources/Textures/_Moffstation/PersonalItems/Wearables/Katzenminer/Siege_Halffloof/siegesmask.rsi/meta.json

```
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
```

Consider licensing the sprite under the newer version of the license:
```suggestion
  "license": "CC-BY-SA-4.0",
```
[Here is an explanation of the differences](https://creativecommons.org/version4/).